src/labs/requests/NewLabRequest.tsx

```diff
@@ -121,7 +121,7 @@ const NewLabRequest = () => {
         <div className="row float-right">
           <div className="btn-group btn-group-lg mt-3">
             <Button className="mr-2" color="success" onClick={onSave}>
-              {t('actions.save')}
+              {t('labs.requests.save')}
```

Comment:
This does not seem to be related to the issue you were working on?

Reply:
Yeah, you are right. This change was related to my previous issue #2301 and it should've been merged into master. I am not sure why the commits are shown up here :(